hugegraph-test/src/main/java/com/baidu/hugegraph/core/AuthTest.java

```diff
@@ -110,9 +110,9 @@ public void testCreateUser() {
         Assert.assertThrows(IllegalArgumentException.class, () -> {
             authManager.createUser(makeUser("tom", "pass1"));
         }, e -> {
-            Assert.assertContains("Can't save user", e.getMessage());
-            Assert.assertContains("that already exists", e.getMessage());
-        });
+                Assert.assertContains("Can't save user", e.getMessage());
```

Comment:
can we keep the origin indentation style?

Reply:
means keeping 12 indents, not 16 ？

Reply:
Yes.
But just now @seagle-yuan said that according to the current style specifications, it may not be possible.

Reply:
according to the latest conclusion, we can keep the original style  lambda alignment